### ISSUE TITLE
Password reset button and options section

### ISSUE THREAD

**Issue by artgancz** [roscisz/TensorHive#199]
There should be provided a button for resetting the password ("Forgot my password" or something like that) as well as the "options" section where I could change my password on my own.

**Comment by micmarty** [comment]
Yep, that's true. We missed some key features that are typical for all web services.
We had to implement most important features first, because of deadlines :)

**Comment by micmarty** [comment]
@filschod Will probably bootstrap whole UI, so that core feature is coming
@micmarty TODO:
- Sending emails when triggered API endpoint
- Generating password reset links
